images/linux/scripts/installers/php.sh

```diff
@@ -10,6 +10,12 @@ source $HELPER_SCRIPTS/document.sh
 source $HELPER_SCRIPTS/os.sh
 
 set -e
+# since 20.04 php is installed from Ubuntu repository
+# see https://github.com/actions/virtual-environments/issues/1084
+if isUbuntu20 ; then
```

Comment:
Why do you split `php.sh` if you can simple add `if` statement?

```
if isUbuntu16 || isUbuntu18; then
    # add repository
    apt-add-repository ppa:ondrej/php -y
    apt-get update
fi
```

Reply:
Using the other repo seems to be a very significant change which should not be handled by "if" branches. Different package names and such. I just created the VM and testing